### DISCUSSION TITLE
Help with device not booting up

### DISCUSSION THREAD

**Discussion by ferbulous** [arendst/Tasmota#19226] [Q&A]
Hi, I flashed tasmota to a homekit downlight (esp8285) but it doesn't seem to boot up/connect to network and I'm only getting green led lit when it's wired to the mains.

![photo1690782211](https://github.com/arendst/Tasmota/assets/24526737/7645978c-45fe-4d60-9e4a-7b3fe9c687e6)
![photo1690782310](https://github.com/arendst/Tasmota/assets/24526737/6752fa83-ea0c-4b17-b9c1-d4c3c157c924)
![photo1690782211 (1)](https://github.com/arendst/Tasmota/assets/24526737/f27838cf-00a8-4b4d-97ad-081b21e18ea7)
![photo1690782211 (2)](https://github.com/arendst/Tasmota/assets/24526737/13040e3a-0ae1-47cd-a0da-a5d64dcf17f6)

```
PS C:\Users\theba\Downloads\Programs\esptool-v3.3.2-win64> esptool.py --port COM5 write_flash -fs 1MB -fm dout 0x0 tasmota_v11.bin
esptool.py v3.3.1
Serial port COM5
Connecting...
Detecting chip type... Unsupported detection protocol, switching and trying again...
Connecting...
Detecting chip type... ESP8266
Chip is ESP8285N08
Features: WiFi, Embedded Flash
Crystal is 26MHz
MAC: 70:03:9f:3d:a1:72
Stub is already running. No upload is necessary.
Configuring flash size...
Flash will be erased from 0x00000000 to 0x0009cfff...
Compressed 641600 bytes to 457011...
Wrote 641600 bytes (457011 compressed) at 0x00000000 in 40.9 seconds (effective 125.6 kbit/s)...
Hash of data verified.
```
I've also tried flashing esphome and restoring the original firmware but it's the still the same problem.
I wasn't able to get any logs from serial 
Anything else I should look into?

**Comment by barbudor** [comment]
To be able to flash you must have connected gpio0 to gnd
If you then get nothing on the serial (115200 bps?), even not the logs of the esp built-in starter, then
- either you have not released gpio0 so the esp can boot from flash
- or you power supply is too weak to allow proper booting


It is important to have a proper 3v3 supply for both flashing and testing on serial. If you provide the 3v3 from the usb/serial adapter then it may be too weak.
You must use a proper adapter that have it's own dedicated 3v3 regulator to feed enough current.
Typical FTDI adapter for example provide a 3v3 only capable to provide a few mA much below the 200-300 mA the esp need for proper booting.

**Reply by ferbulous** [reply]
Hi, for the power supply, i used 3.3v pin from my raspberry pi 4. 
Would that be sufficient?

**Reply by ferbulous** [reply]
Some updates, I used arduino this time for the 3.3v pin. After testing with some esp32 chips, I was able to get log output
But still nothing with the esp8285 device on baud 9600 or 115200

**Reply by Jason2866** [reply]
Still quite sure that flashing is not working correctly. Use the recommended "golden" CH340 from the documentation.

**Reply by ferbulous** [reply]
I was flashing other esp8266/esp32 devices aside this downlight using the same uart and the rest were working just fine.
I’ll try another uart
